packages/okta-angular/src/okta/okta.guard.ts

```diff
@@ -52,7 +52,8 @@ export class OktaAuthGuard implements CanActivate {
     /** 
      * Store the current path
      */
-    this.oktaAuth.setFromUri(state.url);
+    const path = state.url.split(/[?#]/)[0];
```

Comment:
As long as routes are configured as **child routes** (Angular's design decision) this works by removing the `#` and `?` from the URL.